proposal for how the team operates (#215)
rust-lang/rust-project-goals: src/admin/running-the-program.md

# Running the program

> So... somebody suckered you into serving as the owner of the goals program. Congratulations!

Reviewer: ```suggestion
Thank you for helping to own the goals program. Congratulations!
```

(As amusing as the previous wry version was, let's try to be encouraging. :) )
